shelley/chain-and-ledger/formal-spec/leader-value.tex

```diff
@@ -0,0 +1,60 @@
+\section{Leader Value Calculation}
+\label{sec:leader-value-calc}
+
+This section details how we determine whether a node is entitled to lead (under
+the Praos protocol) given the output of its verifiable random function
+calculation.
+
+\begin{figure}
+  \emph{Values associated with the leader value calculations}
+  \begin{equation*}
+  \begin{array}{r@{~\in~}lr}
+    \var{certNat} & \{n | n \in \N, n \in [0,2^{256}]\} & \text{Certified natural value from VRF} \\
+    \var{f} & [0,1] & \text{Active slot coefficient} \\
+    \sigma & [0,1] & \text{Stake proportion}
+  \end{array}
+  \end{equation*}
+\end{figure}
+
+\subsection{Computing the leader value}
+
+The verifiable random function gives us a 32-byte random output. We interpret
+this as a natural number $\var{certNat}$ in the range $[0,2^{256}]$.
```

Comment:
```suggestion
this as a natural number $\var{certNat}$ in the range $[0,2^{256})$.
```

Reply:
Yep, just done that and followed the suggestion to remove the special case.